### DISCUSSION TITLE
Godot Application embed feature

### DISCUSSION THREAD

**Discussion by NeonfireStudio** [godotengine/godot-proposals#8919] [GUI]
There is no feature to embed a application inside in a program made with Godot.Please add this feature after Godot 4.3

**Comment by AThousandShips** [comment]
How would this be accomplished? This is not a trivial feature and some ideas for how to do this would be good

How should this work? Should it be rendered? How should it handle input? Audio?

Not necessarily technical details, but some more details than this

**Comment by Calinou** [comment]
This is already implemented for Android. For iOS, there's an open proposal: https://github.com/godotengine/godot-proposals/issues/1473
